client/src/features/dashboard/components/DatasetsDashboard.tsx

```diff
@@ -0,0 +1,135 @@
+
+
+interface OtherDatasetsButtonProps {
+  totalDatasets: number;
+}
+function OtherDatasetsButton({ totalDatasets }: OtherDatasetsButtonProps) {
+  const dispatch = useDispatch();
+  const history = useHistory();
+  const handleOnClick = (e: React.MouseEvent<HTMLElement>, author: KgAuthor) => {
+    e.preventDefault();
+    dispatch(setAuthor(author));
+    dispatch(setType({ project: false, dataset: true }));
+    history.push(Url.get(Url.pages.searchEntities));
+  };
+  return totalDatasets > MAX_DATASETS_TO_SHOW ?
+    (
+      <div className="d-flex justify-content-center">
+        <Button data-cy="view-my-datasets-btn" className="btn btn-outline-rk-pink"
+          onClick={(e: React.MouseEvent<HTMLElement>) => handleOnClick(e, "user")}>
+          <div className="d-flex gap-2 text-rk-pink">
+            <img src="/frame-pink.svg" className="rk-icon rk-icon-md" />View all my Datasets</div>
+        </Button></div>
+    ) :
+    (<div className="d-flex justify-content-center">
+      <Button data-cy="explore-other-datasets-btn" className="btn btn-outline-rk-pink"
+        onClick={(e: React.MouseEvent<HTMLElement>) => handleOnClick(e, "all")}>
+        <div className="d-flex gap-2 text-rk-pink">
+          <img src="/explore-pink.svg" className="rk-icon rk-icon-md" />Explore other Datasets</div>
+      </Button></div>);
+}
+
+interface DatasetListProps {
+  datasets: any [];
+  gridDisplay: boolean;
+}
+function DatasetListRows({ datasets, gridDisplay }: DatasetListProps) {
+  const items = datasets.map(dataset => mapSearchResultToEntity(dataset));
+
+  return <Fragment>
+    <ListDisplay
+      key="list-datasets"
+      itemsType="dataset"
+      search={null}
+      currentPage={null}
+      gridDisplay={gridDisplay}
+      totalItems={items.length}
+      perPage={items.length}
+      items={items}
+      gridColumnsBreakPoint={{
+        default: 2,
+        1100: 2,
+        700: 2,
+        500: 1
+      }}
+    />
+  </Fragment>;
+}
+
+const MAX_DATASETS_TO_SHOW = 3;
+interface DatasetDashboardProps {
+  userName: string;
+}
+function DatasetDashboard({ userName }: DatasetDashboardProps) {
+  const searchRequest: SearchEntitiesQueryParams = {
+    phrase: "",
+    sort: SortingOptions.DescDate,
+    page: 1,
+    perPage: 3,
+    author: "user",
+    type: {
+      project: false,
+      dataset: true,
+    },
+    userName
+  };
+  const { data, isFetching, isLoading, error } = useSearchEntitiesQuery(searchRequest);
+  const totalUserDatasets = isFetching || isLoading || !data || error ? undefined : data.total;
+  let datasetsToShow;
+  if (isFetching || isLoading) {
+    datasetsToShow = <Loader />;
+  }
+  else if (error || !data) {
+    datasetsToShow = null;
+  }
+  else {
+    datasetsToShow = data.total > 0 ?
+      <DatasetListRows datasets={data.results} gridDisplay={false} />
+      : <p className="rk-dashboard-section-header">You have no current dataset</p>;
```

Comment:
I do not think we need to say "current" here.

```suggestion
      : <p className="rk-dashboard-section-header">You have no datasets</p>;
```

Reply:
Thanks for the review, the text changes make sense to me, I have included it but there may be changes if ui-product requires them although we can apply them later.